model_tests.rb

```diff
@@ -1736,8 +2057,8 @@ def test_model_articulation1_bundle_git_osw
     #puts "standards = #{standards}"
     #puts "workflow = #{workflow}"
 
-    #assert(/0.2.2/.match(standards))
-    #assert(/1.3.2/.match(workflow))
+    assert(/0.2.7/.match(standards))
+    assert(/1.3.2/.match(workflow))
```

Comment:
Actual change in merge, now that I actually set the gem command to install a specific tag